edb/tools/test/__init__.py

```diff
@@ -109,6 +117,88 @@ def test(*, files, jobs, include, exclude, verbose, quiet, debug,
                 f'Error: test path {file!r} does not exist', fg='red')
             sys.exit(1)
 
+    run = functools.partial(
+        _run,
+        include=include,
+        exclude=exclude,
+        verbosity=verbosity,
+        files=files,
+        jobs=jobs,
+        output_format=output_format,
+        warnings=warnings,
+        failfast=failfast,
+    )
+
+    if cov:
+        for pkg in cov:
+            if '\\' in pkg or '/' in pkg:
```

Comment:
nit: os.sep would be enough

Reply:
I had thought about that, but then using `'/'` as separator is kinda also valid on Windows nowadays.